src/main/java/com/example/demo/BusinessComponent/Helpers/Mappers/BusinessMapper.java

```diff
@@ -0,0 +1,38 @@
+package com.example.demo.BusinessComponent.Helpers.Mappers;
+
+import org.springframework.stereotype.Component;
+
+import com.example.demo.BusinessComponent.API.DTOs.BusinessDTO;
+import com.example.demo.BusinessComponent.API.DTOs.CreateBusinessDTO;
+import com.example.demo.BusinessComponent.Domain.Entities.Business;
+import com.example.demo.BusinessComponent.Domain.Entities.User;
+import com.example.demo.BusinessComponent.Helpers.Mappers.Interfaces.IBusinessMapper;
+
+@Component
+public class BusinessMapper implements IBusinessMapper {
+
+  @Override
+  public BusinessDTO businessToBusinessDTO(Business business) {
+    return BusinessDTO
+      .builder()
+      .id(business.getId())
+      .name(business.getName())
+      .ownerId(business.getOwner().getId())
+      .address(business.getAddress())
+      .phoneNumber(business.getPhoneNumber())
+      .emailAddress(business.getEmailAddress())
+      .build();
+  }
+
+  @Override
+  public Business createBusinessDTOToBusiness(CreateBusinessDTO createBusinessDTO) {
```

Comment:
optional, but i feel like if you are already using a BusinessMapper, methods could have more straightforward names such as modelToDTO or mapToDTO

Reply:
Gal jei mapper'is jau pavadintas business, uztenka naudot POST/GET/UPDATE ir etc. nes jau business mapper turi business name? Aij nezinau